ui/src/pages/topics/[id]/index.tsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import Head from 'next/head';
+
+import * as OutlineIcons from '@heroicons/react/24/outline';
+
+import * as api from '@api';
+import * as Assets from '@assets';
+import * as Components from '@components';
+import * as Config from '@config';
+import * as Helpers from '@helpers';
+import * as Interfaces from '@interfaces';
+import * as Layouts from '@layouts';
+import * as Stores from '@stores';
+import * as Types from '@types';
+
+export const getServerSideProps: Types.GetServerSideProps = async (context) => {
+    const id = context.query.id;
+    let topic: Interfaces.Topic | null = null;
+    let error: string | null = null;
+
+    const token = Helpers.getJWT(context);
+
+    try {
+        const response = await api.get(`${Config.endpoints.topics}/${id}`, token);
+        topic = response.data;
+    } catch (err) {
+        const { message } = err as Interfaces.JSONResponseError;
+        error = message;
+    }
+
+    if (!topic || error) {
+        return {
+            notFound: true
+        };
+    }
+
+    return {
+        props: {
+            topic
+        }
+    };
+};
+
+type Props = {
+    topic: Interfaces.Topic;
+};
+
+const Topic: Types.NextPage<Props> = (props): React.ReactElement => {
+    const user = Stores.useAuthStore((state) => state.user);
+
+    const topic = props.topic;
+    const showChildren = Boolean(topic.children.length);
+    const showParents = Boolean(topic.parents.length);
+    return (
+        <>
+            <Head>
+                <meta name="description" content={Config.values.topicDescription} />
+                <link rel="canonical" href={`${Config.urls.viewTopic.canonical}/${topic.id}`} />
+                <title>{`${topic.title} - ${Config.urls.viewTopic.title}`}</title>
+            </Head>
+            <Layouts.Topic fixedHeader={false}>
+                <section className="col-span-9 text-grey-800 dark:text-grey-100">
+                    <header className="border-b border-grey-200">
```

Comment:
is the border-b needed here?

Reply:
This border is needed as there is a border between the header part and the description in the mockup.